src/NuGet.Core/NuGet.Packaging/Rules/DependenciesGroupsForEachTFMRule.cs

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
why is github claiming this is a new file in the branch? it already exists in dev, right?

Reply:
maybe try rebasing onto the latest dev. that might fix it up.